BUILDING.md

```diff
@@ -8,12 +8,12 @@ This repo includes all of Kubernetes history so `--depth 1` will avoid most of t
 
 To build the full release binary run `make` and that will create `./dist/artifacts/k3s`.
 
-Optionally to build the binaries without running linting or building docker images:
+Optionally to build the binaries using local go environment without running linting or building docker images:
 ```sh
 ./scripts/download && ./scripts/build && ./scripts/package-cli
 ```
 
-For development, you just need go 1.12 and a proper GOPATH.  To compile the binaries run:
+For development, you just need go 1.12+ and a proper GOPATH.  To compile the binaries run:
```

Comment:
actually, we're on 1.13, so this should be bumped to 1.13

Reply:
I think this is okay, we just moved to 1.13 but 1.12 should still be supported from a dev/building perspective, especially since we still vendor in modules.

Reply:
Yep, that's why I said "1.12+", it include 1.12/1.13 and the new version for future use.